versions.json

```diff
@@ -1,9 +1,9 @@
 [
     {
-        "version" : "1.42.0"
+        "version" : "1.42.1"
     },
     {
-        "version" : "2.0.0-preview005",
-        "eol" : "2023-07-26"
+        "version" : "2.0.0-preview007",
```

Comment:
do we need this? I thought previews does not need any LTS/EOL

Reply:
Yes, it was a miss from the prior release. This is used for CSS to look up in case if customer contact in.